src/scss/_header.scss

```
@@ -307,7 +307,14 @@
     }
 
     .registry-hero-title {
-        @apply flex items-center;
+        @apply flex items-center flex-wrap;
+
+        // We apply flex-wrap by default so that the search box
+        // wraps to the next line with full-width.
+        // We don't need that for medium+ screens.
+        @screen md {
+            @apply flex-nowrap;
+        }
```

This is okay, but a more mobile-first way of thinking about it would be not to use flex at all by default, and `@apply flex items-center` at medium and up:

```suggestion
        @screen md {
            @apply flex items-center;
        }
```

Verified locally just to make sure it did what I thougth it might:

https://user-images.githubusercontent.com/274700/137540809-22621ba7-a996-49fa-9f0f-90ebed06a1ca.mov

---

Ah sweet! I love it when we can reduce the number of rules we need to use.